tests/pytorch/test_onnx_export.py

```diff
@@ -311,7 +309,7 @@ def forward(self, inp):
 @pytest.mark.parametrize(
     "precision,     atol", [
     [torch.float32, 1e-7],
-    [torch.float16, 2e-3]
+    [torch.float16, 1e-7]
```

Comment:
Looks like this is too strict, let's dial it back a bit

Reply:
Fixed